.travis.yml

```diff
@@ -9,6 +9,13 @@ rvm:
   - 2.3.0
   - ruby-head
   - rbx-2
+  - jruby-9.0.4.0
+  - jruby-9.0.5.0
```

Comment:
We could also maybe consider using an `include` so that we don't run jrubies against all rails version.